lib/chercheville.ex

```diff
@@ -3,23 +3,13 @@ defmodule ChercheVille do
   ChercheVille is an Elixir service allowing to search cities based on data
   from [GeoNames](http://www.geonames.org/).
 
-  ## Adding ChercheVille to your project
-  
-
-      defp deps do
-        [
-          # ...
-          {:chercheville, "~> 0.1.0"}
-        ]
-      end
-
   ## Preparing the database
 
   ChercheVille requires a PostgreSQL database with these extensions enabled:
 
```

Comment:
I find _cities=#_ here a bit confusing.  I think it would be clearer to put 'login to cities database and then run:
`
CREATE EXTENSION postgis
CREATE EXTENSION unaccent;
CREATE EXTENSION pg_trgm;
`